### PR TITLE
Fixed auth response for google login provider #365

### PR DESCRIPTION
**Pull request by bhorpooja**
Fixed AuthResponse issue for google login provider

### CODE REVIEW

File: projects/lib/src/entities/social-user.ts [hunk 1]
```diff
@@ -10,6 +10,7 @@ export class SocialUser {
 
   idToken: string; // Reference https://developers.google.com/identity/sign-in/web/backend-auth
   authorizationCode: string; // Reference https://developers.google.com/identity/sign-in/web/reference#googleauthgrantofflineaccessoptions
+  authResponse: any; // get complete authResponse object
```
Comment (jaibatrik) [inline projects/lib/src/entities/social-user.ts:13]: No need of this, the response field is meant for the auth response.

File: projects/lib/src/providers/google-login-provider.ts [hunk 2]
```diff
@@ -52,13 +52,15 @@ export class GoogleLoginProvider extends BaseLoginProvider {
...
 
         const profile = this.auth2.currentUser.get().getBasicProfile();
+        const authResponse = this.auth2.currentUser.get().getAuthResponse();  // get complete authResponse object
         user.id = profile.getId();
         user.name = profile.getName();
         user.email = profile.getEmail();
         user.photoUrl = profile.getImageUrl();
         user.firstName = profile.getGivenName();
         user.lastName = profile.getFamilyName();
         user.response = profile;
```
Comment (jaibatrik) [inline projects/lib/src/providers/google-login-provider.ts:62]: Change this to return `authResponse` instead. We'll treat it as backward incompatible bug fix.

**Review by jaibatrik — CHANGES_REQUESTED** [review]

File: projects/lib/src/providers/google-login-provider.ts (hunk 2 above)
Comment (bhorpooja) [reply projects/lib/src/providers/google-login-provider.ts:62]: Ok I will make this change

**Comment by bhorpooja** [comment]
@jaibatrik  As you suggested, I updated it. Can you please look it into priority or update this code from your end with any changes if required. 

Thank you @jaibatrik

**Review by jaibatrik — APPROVED** [review]

**Pull request merged**